Modules/_asynciomodule.c

```diff
@@ -3415,16 +3409,12 @@ PyInit__asyncio(void)
         return NULL;
     }
```

Comment:
Maybe add a short comment explaining that PyModule_AddType() initializes types for us:

```
/* FutureType and TaskType are made ready by PyModule_AddType() calls below. */
```

Reply:
Thanks for the suggestion :)